cmd/lotus-shed/datastore-vlog.go

```diff
@@ -34,12 +34,13 @@ var datastoreVlog2CarCmd = &cli.Command{
 			Usage: "datastore prefix",
 			Value: "/blocks/",
 		},
+		&cli.Uint64Flag{Name: "max-size", Value: 32000},
 	},
 	ArgsUsage: "[vlog...]",
 	Action: func(cctx *cli.Context) error {
 		ctx := cctx.Context
 
-		maxSz := uint64(31 << 30)
+		maxSz := uint64(cctx.Uint64("max-size") << 20)
```

Comment:
Are we able convert a blockstore bigger than 31 << 30? If not, we should add a check here to make sure the input is less than 31 << 10

Reply:
This can can do that, it's just usually not useful as in most cases you'll want your cars to be 32/64ish GiB